src/Core/Extensions/EnumExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace QuantumCore.Extensions;
+
+public static class EnumExtensions
```

Comment:
I think "Extensions" is a bad suffix here. It does not contain any extension methods. It's more of a utility?